2-mandatory/4-restaurant.js

```diff
@@ -19,10 +19,33 @@ const MENU = {
   falafel: 7.25,
 };
 
-let cashRegister = {
-  // write code here
+// let cashRegister = {
+//   orderBurger: function (balance) {
+//     const balanceAfterOrder = balance - MENU.burger;
+//     return balanceAfterOrder >= 0 ? balanceAfterOrder : balance;
+//   },
+//   orderFalafel: function (balance) {
+//     const balanceAfterOrder = balance - MENU.falafel;
+//     return balanceAfterOrder >= 0 ? balanceAfterOrder : balance;
+//   },
+// };
+
+// or cleaner code:
+
+const checkAndCountBalance = (bal, itemOfMenu) => {
```

Comment:
I really like that you've separated out the shared business logic into a function 👍